include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -502,16 +567,20 @@ class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Normalize,Clamp,Sw
 								}
 							}
 						}
+
+						double* phaseSupportLUTPixel = (double*)state->phaseSupportLUT->getPointer();
```

Comment:
here you could compute an offset into the scratch

Reply:
Resolved.